execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -214,8 +214,8 @@ impl TransactionGenerator {
         TransactionFactory::new(ChainId::test())
             .with_transaction_expiration_time(300)
             .with_gas_unit_price(100)
-            // TODO(Gas): double check if this is correct
-            .with_max_gas_amount(100_000)
+        // TODO(Gas): double check if this is correct
+        // .with_max_gas_amount(100_000)
```

Comment:
cc @vgao1996

Reply:
I am just removing an override that makes it smaller than max.